todo_app/app.py

```diff
@@ -1,30 +1,47 @@
 from flask import Flask, render_template, request, redirect
-from todo_app.data.session_items import get_items, add_item, remove_item_by_id, get_item_by_id, save_item
+from todo_app.data.models.item import Status
+from todo_app.data.trello_items_service import (
+    get_items,
+    add_item,
+    remove_item_by_id,
+    get_item_by_id,
+    save_item,
+)
 
 from todo_app.flask_config import Config
 
 app = Flask(__name__)
 app.config.from_object(Config())
 
 
+@app.context_processor
+def inject_status_enum():
+    return dict(Status=Status)
```

Comment:
I think this is doing exactly what you're expecting, and is a nice approach, albeit one we might be able to avoid by handling the logic within the Item class (which will tie in a bit more with the ViewModel approach we'll consider in the next module); e.g. we could have a method on the `Item` class that allows us to ask an item directly whether it is complete:
```
class Item:
    status: Status

    ....

    @property
    def is_complete(self):
        return self.status == Status.Completed
```

Which you can then use directly in the Jinja (`{% if item.is_complete %}`)

Reply:
This definitely looks cleaner than my implementation and I like the idea of keeping as much logic in the python side of the code as possible 😄